src/test/java/com/wikia/webdriver/testcases/discussions/PostsList.java

```diff
@@ -31,7 +33,21 @@ public void anonCanSortPostsListOnMobile() {
     userCanSortPostsList();
   }
 
-  /** ANONS ON DESKTOP SECTION */
+  @Test
+  @Execute(browserSize = MOBILE_RESOLUTION, asUser = User.ANONYMOUS)
+  public void anonCanViewMorePosts() {
```

Comment:
please, follow up one naming convention:
`loggedInUser...OnDesktop`
`anonUser...OnMobile`
but I suggest as @ludwikkazmierczak says to use 
what needs to happen --> what did happen
`anonUserOnMobileCanViewMorePosts` (anonUserOnMobile --> CanViewMorePosts) itp.